src/main/java/uk/gov/hmcts/reform/prl/models/c100rebuild/C100RebuildChildDetailsElements.java

```diff
@@ -22,4 +22,6 @@ public class C100RebuildChildDetailsElements {
     private String childrenSubjectOfProtectionPlan;
     @JsonProperty("cd_children")
     private List<ChildDetail> childDetails;
+    @JsonProperty("cd_otherChildren")
+    private List<ChildDetail> otherChildrenDetails;
```

Comment:
@saranyavinayagam8 Are we not capturing 'cd_hasOtherChildren' yes/no field ?

Reply:
added yes/no field and otherChildrenDetails in separate class